app/views/notifications/shared/_comment_box.html.erb

```diff
@@ -15,11 +15,11 @@
         <img class="reacted-emoji" src="<%= asset_path("emoji/emoji-one-heart.png") %>" alt="Favorite heart button" />
       </button>
       <% if context == "moderation" && current_user.has_role?(:trusted) %>
-        <button class="reaction-button <%= Reaction.cached_any_reactions_for?(notification.mocked_object("comment"), current_user, "thumbsdown") ? "reacted" : "" %>" data-reactable-id="<%= json_data["comment"]["id"] %>" data-category="thumbsdown" data-reactable-type="Comment">
+        <button class="reaction-button <%= Reaction.cached_any_reactions_for?(notification.mocked_object("comment"), current_user, "thumbsdown") ? "reacted" : "" %>" data-reactable-id="<%= json_data["comment"]["id"] %>" data-category="thumbsdown" data-reactable-type="Comment" title="thumbsdown">
```

Comment:
These titles make it easier to find button with capybara

Reply:
could we split this in multiple lines?

Reply:
Yep, updated!